src/features/Classes/EnrollStudent/index.jsx

```diff
@@ -47,20 +47,47 @@ const EnrollStudent = ({ isOpen, onClose, queryClassId }) => {
     try {
       setLoading(true);
       const response = await handleEnrollments(formData, queryClassId);
+      const validationEmailList = response?.data?.results;
+      const messages = await getMessages();
+      let validEmails = [];
+      let invalidEmails = [];
+      let textToast = '';
 
       /**
        * This is because the service that checks the enrollment status is a different
        * endpoint, and that endpoint always returns a status 200, so the error cannot be
        * caught with a .catch.
        */
-      if (response?.data?.results[0]?.tags === 'error') {
-        setToastMessage(decodeURIComponent(response?.data?.results[0]?.message));
+      if (messages?.data?.results[0]?.tags === 'error') {
+        setToastMessage(decodeURIComponent(messages?.data?.results[0]?.message));
         setShowToast(true);
 
         return onClose();
       }
 
-      setToastMessage('Email invite has been sent successfully');
+      validationEmailList.map(email => {
```

Comment:
Is necessary `.map` here?

I think this expression could be more simple, but you decide if you see it necessary.
```js
      validationEmailList.forEach(({ invalidIdentifier, identifier }) => {
        (invalidIdentifier ? invalidEmails : validEmails).push(identifier);
      });
```